### PR TITLE
update for template

### PR DESCRIPTION
**Pull request by at-uytran**
image

### CODE REVIEW

**Comment by at-uytran** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -1,33 +1,47 @@
...
-        <% else %>
-          <li><%= link_to t(".log_in"), login_path %></li>
-        <% end %>
-      </ul>
-    </div>
-    <div class="clearfix"> </div>
-  </div>
-</div>
+  <header class="main-header">
+    <!-- Logo -->
```
Comment (tranducquoc) [inline app/views/layouts/_header.html.erb]: tất cả nhứng remarrk này remove hết đi

**Comment by tranducquoc** [comment]
commented
comment + conflict

**Comment by at-uytran** [comment]
ready

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -1,9 +1,6 @@
-<div class="footer">
-  <div class="container">
-      <div class="col-md-3 agile_footer_grids_w3_footer">
-        <div class="w3_footer_grid_bottom">
-          <h4><%= t ".des" %></h4>
-        </div>
-      </div>
-  </div>
-</div>
+  <footer class="main-footer">
```
Comment (tranducquoc) [inline app/views/layouts/_footer.html.erb]: indent

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -1,33 +1,41 @@
...
-              current_user %>
-          <span class="caret"></span></a>
-          <div class="mega-dropdown-menu">
-            <div class="w3ls_vegetables">
-              <ul class="dropdown-menu drp-mnu">
-                <li><%= link_to t(".profile"), current_user %></li>
-                <li class="divider"></li>
-                <li>
-                  <%= link_to t(".log_out"), logout_path, method: :delete %>
+  <header class="main-header">
```
Comment (tranducquoc) [inline app/views/layouts/_header.html.erb]: indent

File: app/views/supervisor/layouts/_footer.html.erb [hunk 4]
```diff
@@ -1,9 +1,6 @@
-<div class="footer">
-  <div class="container">
-      <div class="col-md-3 agile_footer_grids_w3_footer">
-        <div class="w3_footer_grid_bottom">
-          <h4><%= t ".des" %></h4>
-        </div>
-      </div>
-  </div>
-</div>
+  <footer class="main-footer">
```
Comment (tranducquoc) [inline app/views/supervisor/layouts/_footer.html.erb]: indent

File: app/views/supervisor/layouts/_header.html.erb [hunk 5]
```diff
@@ -1,37 +1,41 @@
...
-        </li>
-        <% else %>
-          <li><%= link_to t(".log_in"), login_path %></li>
-        <% end %>
-      </ul>
-    </div>
-    <div class="clearfix"> </div>
-  </div>
-</div>
+  <header class="main-header">
```
Comment (tranducquoc) [inline app/views/supervisor/layouts/_header.html.erb]: indent

File: app/views/supervisor/shared/_sidebar.html.erb [hunk 6]
```diff
@@ -1,16 +1,25 @@
...
-      <li><%= link_to t("static_pages.home.title"), root_path %></a></li>
-      <li><%= link_to t("courses.title"), supervisor_courses_path %></a></li>
-      <% if logged_in? %>
-        <li><%= link_to t(".users"), supervisor_users_path %></li>
-        <li><%= link_to t(".Subject"), supervisor_subjects_path %></li>
-      <% end %>
-    </ul>
-  </div>
-</nav>
+  <aside class="main-sidebar">
```
Comment (tranducquoc) [inline app/views/supervisor/shared/_sidebar.html.erb]: indent

**Comment by tranducquoc** [comment]
commented

**Comment by at-uytran** [comment]
ready

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**